addons/centered/ember.d.ts

```diff
@@ -0,0 +1,2 @@
+import centered from './dist/ember';
```

Comment:
These files need to be added to the files allow-list in package.json

Reply:
why are we disallowing them in the first place? is an optimization? seems like it's going to cause a bunch of headaches. are there other benefits?

Reply:
`@storybook/addon-centered` doesn't actually uses "files" so I think all the files are already included

Reply:
@shilman there are 2 ways to make npm not publish everything:
- explicit not-allowed list
- explicit allowed list

It's true that an allow-list is slightly more cumbersome but:
https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d

.npmignore overrides the .gitignore, which means they have be kept in sync now.

I think having an allow list is slightly more robust.

I'll work on the publishing test I mentioned hopefully soon, so we can be sure the packages work after publishing.

Reply:
@hipstersmoothie it should though?

Reply:
I've never used npm ignores so it seems good not to from that article. I looked again and it seems there is a file list already so I could just add `*.d.ts`

Reply:
why don't we want to publish everything? is an optimization? seems like it's going to cause a bunch of headaches.

Reply:
We want to publish the minimum possible to optimise people's install-time and disk-usage.

Reply:
How much time/space does it save?